app/src/main/java/com/xmartlabs/scasas/criminalintent/ui/crime/list/CrimeListFragment.java

```diff
@@ -50,14 +65,69 @@ public void onResume() {
     updateUI();
   }
 
+  @Override
+  public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+    super.onCreateOptionsMenu(menu, inflater);
+    inflater.inflate(R.menu.fragments_crime_list, menu);
+    MenuItem subtitleItem = menu.findItem(R.id.menu_item_show_subtitle);
+    subtitleItem.setTitle(subtitleVisible ? R.string.hide_subtitle : R.string.show_subtitle);
+  }
+
+  @Override
+  public boolean onOptionsItemSelected(MenuItem item) {
+    switch (item.getItemId()) {
+      case R.id.menu_item_new_crime:
+        Crime crime = Crime.builder()
+            .id(generateUniqueIdentifier())
+            .date(new Date())
+            .solved(false)
+            .title("")
+            .build();
+        CrimeController.getInstance().insertCrime(crime);
+        startActivity(getIntent(crime));
+        return true;
+      case R.id.menu_item_show_subtitle:
+        subtitleVisible = !subtitleVisible;
+        getActivity().invalidateOptionsMenu();
+        updateSubtitile();
+        return true;
+      default:
+        return super.onOptionsItemSelected(item);
+    }
+  }
+
+  private Intent getIntent(Crime crime) {
```

Comment:
Also, i believe this should be `@NonNull` since the `CrimePagerActivity` expects one